Automattic/sensei
# Add life coach pattern

Part of #5245

### Changes proposed in this Pull Request

* It includes a Life Coach course pattern.
  * I needed to tweak a little mainly the spaces, in order to make it work in an acceptable way in all the themes.

### Testing instructions


* Create a new course.
* Navigate through the wizard, and choose the Life Coach pattern.
* Check it in the main themes, and make sure it's working properly.


### Screenshot

---
#### Twenty Twenty-Two
---

Twenty Twenty-Two - Editor
![tt2-life-coach-editor](https://user-images.githubusercontent.com/876340/173101659-d7c986c0-0da8-4a70-9ad7-19e636afdcda.jpg)

Twenty Twenty-Two - Frontend
![tt2-life-coach-frontend](https://user-images.githubusercontent.com/876340/173101700-79efcdad-e6a6-4cc4-b33b-af159879d5df.jpg)

---
#### Twenty Twenty-One
---

Twenty Twenty-One - Editor
![tt1-life-coach-editor](https://user-images.githubusercontent.com/876340/173102199-ead12c86-cd98-4498-b8c8-391e46ee86c1.jpg)

Twenty Twenty-One - Frontend
![tt1-life-coach-frontend](https://user-images.githubusercontent.com/876340/173102172-7449b6e3-c589-461f-90c7-cc4359e27c67.jpg)

---
#### Twenty Twenty
---

Twenty Twenty - Editor
![tt-life-coach-editor](https://user-images.githubusercontent.com/876340/173102456-08bbe04e-1841-48b1-b1ff-0bf865ede55b.jpg)

Twenty Twenty - Frontend
![tt-life-coach-frontend](https://user-images.githubusercontent.com/876340/173102489-cbc98405-bb69-4a25-b216-7b4e3cc9d818.jpg)

---
#### Divi
---

Divi - Editor
![divi-life-coach-editor](https://user-images.githubusercontent.com/876340/173102528-14b1c45c-fc7f-47c3-bf88-e0d0dcfba9dc.jpg)

Divi - Frontend
![divi-life-coach-frontend](https://user-images.githubusercontent.com/876340/173102543-1cb8d6c1-d6fc-4e26-85cc-7ffa40788ecf.jpg)

---
#### Astra
---

Astra - Editor
![astra-life-coach-editor](https://user-images.githubusercontent.com/876340/173102578-891806ca-1e06-4b0e-a413-fdec64940a03.jpg)

Astra - Frontend
![astra-life-coach-frontend](https://user-images.githubusercontent.com/876340/173102570-2478f897-52c0-4d96-af8a-8094d043b919.jpg)

---
#### GeneratePress
---

GeneratePress - Editor
![generatepress-life-coach-editor](https://user-images.githubusercontent.com/876340/173102611-340c1a24-c25f-4fec-878b-a89af54597f7.jpg)

GeneratePress - Frontend
![generatepress-life-coach-frontend](https://user-images.githubusercontent.com/876340/173102601-67957a96-97bc-44f1-9c66-69e2d02ebc35.jpg)

---
#### Storefront
---

Storefront - Editor
![storefront-life-coach-editor](https://user-images.githubusercontent.com/876340/173102631-213a16ae-a55d-4a26-8d43-7251e942d0fc.jpg)

Storefront - Frontend
![storefront-life-coach-frontend](https://user-images.githubusercontent.com/876340/173102640-7db4764c-628b-440c-8c39-c34373cf2b55.jpg)

---
#### OceanWP
---

Notice that OceanWP has an issue with an image position. It's because of the Media & Text block inside the Group group with full width. I didn't want to change that to avoid a bigger impact in the other themes.

OceanWP - Editor
![oceanwp-life-coach-editor](https://user-images.githubusercontent.com/876340/173102654-b10f4bc0-0ce4-43a8-a800-a61682a45cdf.jpg)

OceanWP - Frontend
![oceanwp-life-coach-frontend](https://user-images.githubusercontent.com/876340/173102650-74691d2b-d17d-45c2-86ef-a85d922a79e2.jpg)
